feature/client/src/commonMain/kotlin/com/mifos/feature/client/clientEditProfile/ClientProfileEditScreen.kt

```
@@ -206,6 +206,16 @@ private fun ClientProfileEditDialogs(
             )
         }
 
+        is ClientProfileEditState.DialogState.Success -> {
+            MifosBasicDialog(
+                title = "Success",
+                content = { Text("Your profile image has been updated successfully!") },
+                confirmText = "Done",
```

use Success and Failure Dilaogs instead of Basic Dialog.

---

This one

MifosStatusDialog

which accepts success and Failure state and also move strings to string resources

---

I am done with the changes